WordPress/src/main/java/org/wordpress/android/ui/DeeplinkWebViewActivity.kt

```diff
@@ -0,0 +1,52 @@
+package org.wordpress.android.ui
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.webkit.WebResourceRequest
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import org.wordpress.android.R
+import org.wordpress.android.ui.accounts.LoginMagicLinkInterceptActivity
+
+class DeeplinkWebViewActivity : WebViewActivity() {
```

Comment:
If we decide to handle `mbar` links instead of `bar` ones, as we have been discussing recently, then I think we should be able to get rid of this class, right?